production/main.R

```diff
@@ -6,7 +6,7 @@ suppressPackageStartupMessages(library(R6))
 suppressPackageStartupMessages(library(tryCatchLog))
 suppressPackageStartupMessages(library(futile.logger))
 suppressPackageStartupMessages(library(googlesheets4))
-gs4_auth("ucla.law.covid.staff@gmail.com")
+gs4_auth("ucla.law.covid.staff@gmail.com", scopes = "https://www.googleapis.com/auth/drive")
```

Comment:
Can you limit this pull request to just be the work related to the diagnostic move?